manual/source/mpes-structure.rst

```diff
@@ -28,3 +28,6 @@ Here's a list of application definitions related to photoemission spectroscopy:
 
     :ref:`NXmpes`:
        A universal application definition with minimal metadata requirements for describing all photoemission experiments.
+
+    :ref:`NXxps`:
+       An appdef for XPS/UPS measurements
```

Comment:
What is UPS spell out abbreviations at first occurrence, maybe you have?

Reply:
Spelled out the abbrevations, thanks!